conan/tools/google/bazel.py

```diff
@@ -40,5 +39,6 @@ def build(self, args=None, label=None):
     def _get_bazel_project_configuration(self):
         toolchain_file_content = load_toolchain_args(self._conanfile.generators_folder,
                                                      namespace=self._namespace)
-        self._bazel_config = toolchain_file_content.get("bazel_config")
+        configs = toolchain_file_content.get("bazel_configs")
+        self._bazel_config = configs.split(",") if configs else []
```

Comment:
a bit dirty, storing a list type with a comma-separated string.

Reply:
Yes, this is a bit dirty. I don't recall why exactly we changed our previous json toolchain_file for the INI one, but it seems the INI file is being a bit limited?

Reply:
yes.

Reply:
Taking a second look it looks like the thing to store here are directly the command line arguments to call bazel: `--config foo --config bar` like we've done with others

Reply:
I got the original issue: https://github.com/conan-io/conan/issues/9130

Reply:
I'm checking the ConfigParser documentation and I saw the `converters` keyword. Perhaps, we could add something like this:
```python
    toolchain_config = configparser.ConfigParser(converters={"list": lambda x: x.split(",")})
    # ...
    configs = toolchain_file_content.getlist("bazel_configs", fallback=[])
```
So, adding that `converters` keyword into the `load_toolchain_args` function, we could pass our own `converters` like that one:
```python
def load_toolchain_args(generators_folder=None, namespace=None, converters=None):
```

Reply:
I am still not very convinced. I'd like to try the CMakePresets.json-like file approach, it would be a great thing for modern CMake users, it is already demanded. But it is a json, if we cannot store things in json, this won't be feasible at all? Maybe things can be stored un-scaped in the json?